x-pack/test_serverless/functional/test_suites/common/management/advanced_settings.ts

```diff
@@ -7,18 +7,33 @@
 
 import expect from '@kbn/expect';
 import { ALL_COMMON_SETTINGS } from '@kbn/serverless-common-settings';
+import * as settings from '@kbn/management-settings-ids';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
+const editorSettings = new Set<string>([
+  settings.BANNERS_TEXT_CONTENT_ID,
+  settings.DATE_FORMAT_SCALED_ID,
+  settings.ML_ANOMALY_DETECTION_RESULTS_TIME_DEFAULTS_ID,
+  settings.NOTIFICATIONS_BANNER_ID,
+  settings.TIMEPICKER_TIME_DEFAULTS_ID,
+  settings.TIMEPICKER_QUICK_RANGES_ID,
+  settings.SECURITY_SOLUTION_REFRESH_INTERVAL_DEFAULTS_ID,
+  settings.SECURITY_SOLUTION_TIME_DEFAULTS_ID,
+  settings.SECURITY_SOLUTION_RULES_TABLE_REFRESH_ID,
+  settings.SECURITY_SOLUTION_IP_REPUTATION_LINKS_ID,
+]);
+export const isEditorFieldSetting = (settingId: string) => editorSettings.has(settingId);
+
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const testSubjects = getService('testSubjects');
-  const pageObjects = getPageObjects(['common']);
+  const pageObjects = getPageObjects(['svlCommonPage', 'common']);
   const browser = getService('browser');
   const retry = getService('retry');
 
-  // Skip until we enable the Advanced settings app in serverless
-  describe.skip('Common advanced settings', function () {
+  describe('Common advanced settings', function () {
     before(async () => {
-      await pageObjects.common.navigateToApp('advancedSettings');
+      await pageObjects.svlCommonPage.login();
```

Comment:
Tests should also logout when they're done:
```
    after(async () => {
      await svlCommonPage.forceLogout();
    });
```
(same for the other tests, I won't repeat the comment)

Reply:
@pheyos If this is the only comment blocking this PR, could you approve pending that change?  We want to ship this relatively quickly.  Let me know if you want to wait until this change is in the commit and we can ping you.

Reply:
Thanks @pheyos! Added the change with https://github.com/elastic/kibana/pull/167383/commits/6f50efa8baf6b4db1fb68d0f8e19e19c85cd458a.